library/src/main/java/com/chuckerteam/chucker/internal/data/room/ChuckerDatabase.kt

```diff
@@ -9,7 +9,7 @@ import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
 @Database(entities = [HttpTransaction::class], version = 6, exportSchema = false)
 internal abstract class ChuckerDatabase : RoomDatabase() {
 
-    abstract fun transactionDao(): HttpTransactionDao
+    abstract val transactionDao: HttpTransactionDao
```

Comment:
Same here, why change this?

Reply:
Kotlin feature, use`val` replace `fun` in`interface` or `abstract class`.

Reply:
I don't know why would you like to change it. There is nothing wrong or unidiomatic with using a function.

Reply:
Yeah, behaves in the same way. But I think use a field instead of function here is more graceful.

Reply:
This is just opinion based. I don't mind the output code but I don't like to change the code just for cosmetic changes when there are no objective indicators to do so.

Reply:
Another reason, for Java code to access it, we usually use `getTransactionDao()`, but in Kotlin we are suggested to use `transactionDao`. What I modified can be more compatible with Java and Kotlin.

Reply:
This would be true, if this class was exposed. Don't get me wrong, I'm not requesting for you to change it back to a function but it is not something that refines the code. It is just formatting it according to personal preferences.